chips/stm32f4xx/src/dma/mod.rs

```diff
@@ -1,17 +1,18 @@
+use core::fmt::Debug;
```

Comment:
Can we name this something more descriptive than "mod"? I'm not sure what this file contains that is not in the other dma files.

Reply:
Recombined the dma subfolder into a `dma.rs` file